src/nbb/impl/repl.cljs

```diff
@@ -170,8 +170,19 @@
   ([] (repl nil))
   ([_opts]
    (when tty (.setRawMode js/process.stdin true))
-   (js/Promise. (fn [resolve]
-                  (input-loop nil resolve)))))
+   (let [eval-require (fn
+                        [ns-form]
+                        (nbb/eval-require
+                         (list
+                          'quote
```

Comment:
The `'quote` originally is the `require` symbol in `(require '[....])` so it's probably clearer if we name that `require` here as well.